client_lib/client.exs

```diff
@@ -1,5 +1,5 @@
 #! /usr/bin/elixir
-# command = "openssl s_client -connect localhost:41043 -showcerts -msg -servername local -tls1_2 -tlsextdebug -curves secp256k1 -cert device_certificate.pem"
+# command = "openssl s_client -connect localhost:41_043 -showcerts -msg -servername local -tls1_2 -tlsextdebug -curves secp256k1 -cert device_certificate.pem"
```

Comment:
This change in the comment is wrong. The openssl command line would not understand the _ format.